lib/commanded/commands/router.ex

```diff
@@ -507,16 +507,18 @@ defmodule Commanded.Commands.Router do
         @command_opts command_opts
 
         defp do_dispatch(%@command_module{} = command, opts) do
+          alias Commanded.Application
           alias Commanded.Commands.Dispatcher
           alias Commanded.Commands.Dispatcher.Payload
 
           opts = Keyword.merge(@command_opts, opts)
 
           application = Keyword.fetch!(opts, :application)
+          uuid_generator = Application.uuid_generator(application, false) || fn -> nil end
```

Comment:
Maybe `uuid_generator` should return an `fn -> nil end` instead of just `nil` ?

Reply:
Also updated  `Application.uuid_generator!` returning an a zero arity function